src/TockContext.tsx

```diff
@@ -204,17 +205,22 @@ export const tockReducer: Reducer<TockState, TockAction> = (
   return state;
 };
 
-const TockContext: (props: { children?: ReactNode }) => JSX.Element = ({
+const TockContext: (props: {
+  children?: ReactNode;
+  localStorageHistory?: TockLocalStorage;
+}) => JSX.Element = ({
   children,
+  localStorageHistory,
 }: {
   children?: ReactNode;
+  localStorageHistory?: TockLocalStorage;
 }) => {
   const [state, dispatch]: [TockState, Dispatch<TockAction>] = useReducer(
     tockReducer,
     {
       quickReplies: [],
       messages: [],
-      userId: retrieveUserId(),
+      userId: retrieveUserId(localStorageHistory),
```

Comment:
Is it a problem if multiple bots share the same userId ? I know that the userId mecanism is not explicitly explain in the README but adding a prefix in localStorageHistory without knowing that it will affect the key of the userId can be troublesome. I should have name the parameter localStorage instead of localStorageHistory :(

Reply:
Since localStorageHistory was the name used to pass these same parameters to Chat.tsx, I didn't think it wise to use a different name for the same purpose.

Reply:
Concerning userId storage, I don't know the ins and outs on the backend, so I don't know if it's a problem for several bots to share the same userId. When in doubt, I thought it safer not to cause this ambiguity, but if someone can confirm that no side-effects are possible, I can go back and leave the key unchanged.

Reply:
I would say it is indeed preferable to err on the safe side and avoid reusing userId from the local storage. Although in any case, the better practice would be to avoid using localStorage userId entirely and enable cookie-based auth in the TOCK backend.

Reply:
@correi-f is right though, this may cause unexpected behavior from the users' perspective. We should at least document the side effects in the readme, and add a TODO comment to eventually rename the `localStorageHistory` option to simply `localStorage`.